dwave/cloud/auth/flows.py

```diff
@@ -31,11 +31,13 @@
 from dwave.cloud.auth.config import OCEAN_SDK_CLIENT_ID, OCEAN_SDK_SCOPES
 from dwave.cloud.auth.creds import Credentials
 from dwave.cloud.auth.server import SingleRequestAppServer, RequestCaptureAndRedirectApp
-from dwave.cloud.config.models import ClientConfig
 from dwave.cloud.regions import resolve_endpoints
 from dwave.cloud.utils.http import default_user_agent
 from dwave.cloud.utils.logging import pretty_argvalues
 
+if TYPE_CHECKING:
```

Comment:
This is complaining about Python, not the code, but I find this such an awkward flow. I wish there was like a `timport` or something like that.